src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-param-reassign */
```

Comment:
this line according to this issue
https://eslint.org/docs/latest/rules/no-param-reassign